File: app/src/main/java/com/zimax/bussiness/CountryFlag.java
package com.zimax.bussiness;

import com.zimax.R;
import java.util.HashMap;
import java.util.Map;

public class CountryFlag {

    private Map<String, Integer> mapFlagContainer = new HashMap<>();

    public Map<String, Integer> getFlagContainer() {
        return mapFlagContainer;
    }

    public void createFlag() {
        mapFlagContainer.put("AUD", R.drawable.aud);
        mapFlagContainer.put("AMD", R.drawable.amd);
        mapFlagContainer.put("BGN", R.drawable.bgn);
        mapFlagContainer.put("AZN", R.drawable.azn);
        mapFlagContainer.put("BRL", R.drawable.brl);
        mapFlagContainer.put("BYN", R.drawable.byn);
        mapFlagContainer.put("CAD", R.drawable.cad);
        mapFlagContainer.put("CHF", R.drawable.chf);
        mapFlagContainer.put("CNY", R.drawable.cny);
        mapFlagContainer.put("CZK", R.drawable.czk);
        mapFlagContainer.put("DKK", R.drawable.dkk);
        mapFlagContainer.put("EUR", R.drawable.eur);
        mapFlagContainer.put("FBP", R.drawable.gbp);
        mapFlagContainer.put("HKD", R.drawable.hkd);
        mapFlagContainer.put("HUF", R.drawable.huf);
        mapFlagContainer.put("INR", R.drawable.inr);
        mapFlagContainer.put("JPY", R.drawable.jpy);
        mapFlagContainer.put("KGS", R.drawable.kgs);
        mapFlagContainer.put("KRW", R.drawable.krw);
        mapFlagContainer.put("KZT", R.drawable.kzt);
        mapFlagContainer.put("MDL", R.drawable.mdl);
        mapFlagContainer.put("NOK", R.drawable.nok);
        mapFlagContainer.put("RON", R.drawable.ron);
        mapFlagContainer.put("SEK", R.drawable.sek);
        mapFlagContainer.put("SGD", R.drawable.sgd);
        mapFlagContainer.put("TJS", R.drawable.tjs);
        mapFlagContainer.put("TMT", R.drawable.tmt);
        mapFlagContainer.put("TRY", R.drawable.try_f);
        mapFlagContainer.put("UAH", R.drawable.uah);
        mapFlagContainer.put("USD", R.drawable.usd);
        mapFlagContainer.put("UZS", R.drawable.uzs);
        mapFlagContainer.put("XDR", R.drawable.xdr);
        mapFlagContainer.put("ZAR", R.drawable.zar);
        mapFlagContainer.put("GBP", R.drawable.gbp_fl);
        mapFlagContainer.put("PLN", R.drawable.pln_fl);
    }

}

File: app/src/main/java/com/zimax/bussiness/CountryFlag.kt
package com.zimax.bussiness

import com.zimax.R
import java.util.*

class CountryFlag {

    private val mapFlagContainer: MutableMap<String, Int> = HashMap()
    val flagContainer: Map<String, Int>
        get() = mapFlagContainer

    fun createFlag() {
        mapFlagContainer["AUD"] = R.drawable.aud
        mapFlagContainer["AMD"] = R.drawable.amd
        mapFlagContainer["BGN"] = R.drawable.bgn
        mapFlagContainer["AZN"] = R.drawable.azn
        mapFlagContainer["BRL"] = R.drawable.brl
        mapFlagContainer["BYN"] = R.drawable.byn
        mapFlagContainer["CAD"] = R.drawable.cad
        mapFlagContainer["CHF"] = R.drawable.chf
        mapFlagContainer["CNY"] = R.drawable.cny
        mapFlagContainer["CZK"] = R.drawable.czk
        mapFlagContainer["DKK"] = R.drawable.dkk
        mapFlagContainer["EUR"] = R.drawable.eur
        mapFlagContainer["FBP"] = R.drawable.gbp
        mapFlagContainer["HKD"] = R.drawable.hkd
        mapFlagContainer["HUF"] = R.drawable.huf
        mapFlagContainer["INR"] = R.drawable.inr
        mapFlagContainer["JPY"] = R.drawable.jpy
        mapFlagContainer["KGS"] = R.drawable.kgs
        mapFlagContainer["KRW"] = R.drawable.krw
        mapFlagContainer["KZT"] = R.drawable.kzt
        mapFlagContainer["MDL"] = R.drawable.mdl
        mapFlagContainer["NOK"] = R.drawable.nok
        mapFlagContainer["RON"] = R.drawable.ron
        mapFlagContainer["SEK"] = R.drawable.sek
        mapFlagContainer["SGD"] = R.drawable.sgd
        mapFlagContainer["TJS"] = R.drawable.tjs
        mapFlagContainer["TMT"] = R.drawable.tmt
        mapFlagContainer["TRY"] = R.drawable.try_f
        mapFlagContainer["UAH"] = R.drawable.uah
        mapFlagContainer["USD"] = R.drawable.usd
        mapFlagContainer["UZS"] = R.drawable.uzs
        mapFlagContainer["XDR"] = R.drawable.xdr
        mapFlagContainer["ZAR"] = R.drawable.zar
        mapFlagContainer["GBP"] = R.drawable.gbp_fl
        mapFlagContainer["PLN"] = R.drawable.pln_fl
    }
}